apps/src/applab/applab.js

```diff
@@ -449,7 +450,7 @@ function queueOnTick() {
 }
 
 function handleExecutionError(err, lineNumber) {
-  outputError(String(err), ErrorLevel.ERROR, lineNumber);
+  error(String(err), lineNumber);
```

Comment:
I kind of like outputError better as a name here. It makes it a little more clear what the error is being used for.